content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -0,0 +1,165 @@
+
+3. Copy the example `validators.txt` file to the same folder as `rippled.cfg`:
+
+        cp doc/validators-example.txt ~/.config/ripple/validators.txt
+
+    **Warning:** Ripple has designed and [documented](https://ripple.com/dev-blog/decentralization-strategy-update/) a decentralization plan with maximum safety in mind. During the transition, you *should not* modify the `validators.txt` file except as recommended by Ripple. Even minor modifications to your validator settings could cause your server to diverge from the rest of the network and report out of date, incomplete, or inaccurate data. Acting on such data can cause you to lose money.
+
+## 3. Run `rippled`
+
+To run your stock `rippled` server from the executable you built, using the configurations you defined:
+```
+cd build
+./rippled
+```
+
+
+### What to Expect
+
+Once you've run `rippled`, here are excerpts of what you can expect to see in your terminal.
+
+```
+Loading: "/etc/opt/ripple/rippled.cfg"
+Watchdog: Launching child 1
+2017-Dec-15 18:27:37 JobQueue:NFO Auto-tuning to 4 validation/transaction/proposal threads for non-validator.
+2017-Dec-15 18:27:37 Amendments:DBG Amendment C6970A8B603D8778783B61C0D445C23D1633CCFAEF0D43E7DBCD1521D34BD7C3 is supported.
+2017-Dec-15 18:27:37 Amendments:DBG Amendment 4C97EBA926031A7CF7D7B36FDE3ED66DDA5421192D63DE53FFB46E43B9DC8373 is supported.
+...
+2017-Dec-15 18:27:37 ValidatorList:DBG Loading configured trusted validator list publisher keys
+...
+2017-Dec-15 18:27:37 ValidatorList:DBG Loading configured validator keys
+...
+2017-Dec-15 18:27:37 ValidatorSite:DBG Loading configured validator list sites
+...
+2017-Dec-15 18:27:38 LedgerConsensus:NFO Entering consensus process, watching, synced=no
+...
+2017-Dec-15 18:27:40 LedgerConsensus:WRN
+{
+	"accepted" : true,
+	"account_hash" : "183D5235C7C1FB5AE67AD2F6CC3B28F5FB86E8C4F89DB50DD85641A96470534E",
+	"close_flags" : 0,
+	"close_time" : 566677650,
+	"close_time_human" : "2017-Dec-15 18:27:30",
+	"close_time_resolution" : 30,
+	"closed" : true,
+	"hash" : "014816C71F4B9AD5924AFBD7B02E0DF1522C4BC4BC45A69F97F51D4F2234FA10",
+	"ledger_hash" : "014816C71F4B9AD5924AFBD7B02E0DF1522C4BC4BC45A69F97F51D4F2234FA10",
+	"ledger_index" : "2",
+	"parent_close_time" : 0,
+	"parent_hash" : "AB868A6CFEEC779C2FF845C0AF00A642259986AF40C01976A7F842B6918936C7",
+	"seqNum" : "2",
+	"totalCoins" : "100000000000000000",
+	"total_coins" : "100000000000000000",
+	"transaction_hash" : "0000000000000000000000000000000000000000000000000000000000000000"
+}
+...
+2017-Dec-15 18:27:50 NetworkOPs:WRN We are not running on the consensus ledger
+2017-Dec-15 18:27:50 LedgerConsensus:WRN Need consensus ledger 8459443546CCBE4E89A41E1D0C17F24FBEA7B344C1ED6A26292B4C5CD19B4321
+...
+2017-Dec-15 18:38:28 OpenLedger:WRN Offer involves frozen asset
+2017-Dec-15 18:38:31 LedgerConsensus:WRN Offer involves frozen asset
+```
+
+
+## Explore Next Steps
```

Comment:
...such as moving the binary to /usr/bin/rippled and installing a systemd unit for it.

Reply:
Added installing a systemd unit to "Explore Next Steps." Thank you for the suggestion!